backend/src/main/java/com/festago/presentation/MemberTicketController.java

```diff
@@ -61,15 +65,20 @@ public ResponseEntity<MemberTicketResponse> findById(
     }
 
     @GetMapping
-    public ResponseEntity<MemberTicketsResponse> findAll(@Login LoginMember loginMember) {
-        MemberTicketsResponse response = memberTicketService.findAll(loginMember.memberId());
+    public ResponseEntity<MemberTicketsResponse> findAll(
+        @Login LoginMember loginMember,
+        @RequestParam(defaultValue = "0") int page,
+        @RequestParam(defaultValue = "100") int size) {
+        Pageable pageRequest = PageRequest.of(page, size, Sort.by("entryTime").descending());
```

Comment:
```suggestion
        Pageable pageRequest = PageRequest.of(page, size, Sort.by("id").descending());
```
지금 생각했는데 id가 생성 순으로 됥텐데, createAt 대신 사용해도 좋을 것 같네요!
(성능 생각하면 이게 더 좋을지도)